composeApp/src/commonMain/kotlin/sp/bvantur/tasky/login/data/LoginUserDataRequest.kt

```diff
@@ -0,0 +1,12 @@
+package sp.bvantur.tasky.login.data
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class LoginUserDataRequest(
+    @SerialName("email")
+    val email: String,
+    @SerialName("password")
+    val password: String
+)
```

Comment:
Tip, if the property name is already the same the JSON response, then you don’t need the extra `@SerialName` annotations.